recipe/pp-sketchlib-aarch64.patch

```diff
@@ -12,23 +12,36 @@ index 0adda45..a813d93 100644
  # Set paths for non standard lib/ and include/ locations
  if(DEFINED ENV{CONDA_PREFIX})
 diff --git i/src/Makefile w/src/Makefile
-index feac46b..0c13f0c 100644
+index feac46b..a719e39 100644
 --- i/src/Makefile
 +++ w/src/Makefile
-@@ -14,7 +14,6 @@ UNAME_S := $(shell uname -s)
+@@ -11,10 +11,10 @@ else
+ endif
+ 
+ UNAME_S := $(shell uname -s)
++UNAME_M := $(shell uname -m)
  LIBLOC = ${CONDA_PREFIX}
  LDLIBS = -lz -lhdf5_cpp -lhdf5 -lopenblas -lgomp
  ifeq ($(UNAME_S),Linux)
 -	CXXFLAGS+= -m64
  	ifdef PROFILE
  		CXXFLAGS+= -Wl,--compress-debug-sections=none
  	endif
-@@ -29,7 +28,7 @@ CPPFLAGS+=-I"." -I$(LIBLOC)/include -I$(LIBLOC)/include/eigen3
+@@ -29,7 +29,7 @@ CPPFLAGS+=-I"." -I$(LIBLOC)/include -I$(LIBLOC)/include/eigen3
  LDFLAGS+= -L$(LIBLOC)/lib
  CUDA_LDLIBS=-lcudadevrt -lcudart_static $(LDLIBS)
  
 -CUDA_LDFLAGS =-L$(LIBLOC)/lib -L${CUDA_HOME}/targets/x86_64-linux/lib/stubs -L${CUDA_HOME}/targets/x86_64-linux/lib
-+CUDA_LDFLAGS =-L$(LIBLOC)/lib -L${CUDA_HOME}/targets/aarch64-linux/lib/stubs -L${CUDA_HOME}/targets/aarch64-linux/lib
++CUDA_LDFLAGS =-L$(LIBLOC)/lib -L${CUDA_HOME}/targets/$(UNAME_M)-linux/lib/stubs -L${CUDA_HOME}/targets/$(UNAME_M)-linux/lib
```

Comment:
This almost works

However ARM comes in different flavors. The one we typically use is called [SBSA]( https://en.wikipedia.org/wiki/Server_Base_System_Architecture ). So for that case we want `sbsa-linux`

Here is [the full listing of options]( https://github.com/conda-forge/cuda-cudart-feedstock/blob/5332513859c6841f3306563a40556bf699b5a003/recipe/meta.yaml#L8-L10 )